pkg/agent/cniserver/pod_configuration.go

```diff
@@ -87,8 +87,7 @@ func configureContainerAddr(netns ns.NetNS, containerInterface *current.Interfac
 			return err
 		}
 		// Send gratuitous ARP to network in case of stale mappings for this IP address
-		// (e.g. if a previous - deleted - Pod was using the same IP). The GARP is sent to
```

Comment:
@tnqn out of curiosity was that part of the comment actually wrong or did you consider it redundant?

Reply:
It‘s not accurate, it's not only sent to Pod but also gw0 of host, and the latter is the problem of the issue we met. I tried to say "The GARP is sent to all Pods and gateway interface in the subnet." but felt is's well-known to people working on networking, and "sending gratuitous ARP to network" is good enough.